public/app/features/dashboard/components/PanelEditor/PanelEditorTableView.test.tsx

```diff
@@ -0,0 +1,184 @@
+import { act, render, screen } from '@testing-library/react';
+import React, { FC } from 'react';
+import { Provider } from 'react-redux';
+import configureMockStore from 'redux-mock-store';
+import { ReplaySubject } from 'rxjs';
+import { TimeSrvStub } from 'test/specs/helpers';
+
+import {
+  dateTime,
+  EventBusSrv,
+  getDefaultTimeRange,
+  LoadingState,
+  PanelData,
+  PanelPlugin,
+  PanelProps,
+  TimeRange,
+} from '@grafana/data';
+import { getTimeSrv } from 'app/features/dashboard/services/TimeSrv';
+
+import { PanelQueryRunner } from '../../../query/state/PanelQueryRunner';
+import { DashboardModel, PanelModel } from '../../state';
+
+import { PanelEditorTableView, Props } from './PanelEditorTableView';
+
+jest.mock('../../utils/panel', () => ({
+  applyPanelTimeOverrides: jest.fn((panel, timeRange) => ({
+    ...timeRange,
+  })),
+}));
+
+// Mock timeSrv singleton instance
+const timeSrvMock = new TimeSrvStub();
+jest.mock('../../services/TimeSrv', () => ({
+  ...jest.requireActual('../../services/TimeSrv'),
+  getTimeSrv: jest.fn(() => {
+    return timeSrvMock;
```

Comment:
🤔 I am not sure if this is the best way to mock `getTimeSrv`

Reply:
@axelavargas I think a simpler way would be to just use the service setter (`setTimeSrv` from 
 `app/features/dashboard/services/TimeSrv`), and just do the following in `beforeAll`:

```
setTimeSrv(timeSrvMock)
```

Reply:
yes! that's a way better approach, thank you, Dominik 🙏🏾 .  I just commit the suggestion 🙌🏾